apps/package.json

```diff
@@ -34,10 +35,15 @@
   "browser": {
     "blockly": "./build/package/js/blockly.js"
   },
+  "resolutions": {
+    "eshost": "github:pcardune/eshost#f3fa7fa",
+    "test262-compiler": "github:pcardune/test262-compiler#ffa0dff"
```

Comment:
1. I think I might be able to get rid of this resolutions thing actually. It should get taken care of by what is in the yarn.lock file.

2. `eshost` includes a new agent definition for js-interpreter. I have an issue out asking for a better way to add new agents without having them live in the `eshost` repo. Haven't heard back yet :(. `test262-compiler` just has a small (but important) bug fix which I likewise have a pull request out for but haven't heard back yet either.

pull requests in question:
* https://github.com/bterlson/eshost/pull/12
* https://github.com/bterlson/test262-compiler/pull/1